homeassistant/components/onvif/manifest.json

```diff
@@ -3,12 +3,13 @@
   "name": "Onvif",
   "documentation": "https://www.home-assistant.io/components/onvif",
   "requirements": [
-    "onvif-py3==0.1.3",
+    "onvif-zeep-async==0.1.1",
+    "zeep[async]==3.0.0",
     "suds-passworddigest-homeassistant==0.1.2a0.dev0",
     "suds-py3==1.3.3.0"
```

Comment:
should be removed since no longer a dep.

Reply:
Good catch, done!